ui/src/routes/(platform)/+layout.svelte

```diff
@@ -82,14 +84,43 @@
       path: '/link-3'
     }
   ];
+
+  const mapRouteSegments = (route: string): { [segment: string]: string } => {
+    // Split the route into segments
+    const segments = route.split('/').filter((segment) => segment.length > 0);
+
+    // Initialize an empty object to store the mapping
+    const routeMapping: { [segment: string]: string } = {};
+
+    // Initialize a variable to build the link incrementally
+    let link = '';
+
+    // Loop through each segment and build the mapping
+    segments.forEach((segment, index) => {
+      // Add the segment to the link
+      link += (index > 0 ? '/' : '') + segment;
+
+      let capitalizedSegment = `${segment.charAt(0).toUpperCase()}${segment.slice(1, segment.length)}`;
+      routeMapping[capitalizedSegment] = link;
+    });
+
+    return routeMapping;
+  };
+
+  $: navRouteNames = mapRouteSegments($page.url.pathname);
```

Comment:
I'm not sure this is necessary atm. If I'm reading this right, we're creating route names for the Breadcrumbs based on the URL. I think we can do something simpler for now like in this [example](https://svelte.carbondesignsystem.com/components/Breadcrumb).